Source/Core/VideoBackends/OGL/VertexManager.cpp

```diff
@@ -156,8 +158,20 @@ void VertexManager::vFlush()
   // setup the pointers
   nativeVertexFmt->SetupVertexPointers();
 
+  if (!g_Config.backend_info.bSupportsFragmentStoresAndAtomics && ::BoundingBox::active)
+  {
+    glEnable(GL_STENCIL_TEST);
+    OGL::BoundingBox::stencil_updated = true;
```

Comment:
Reading this again, it's somewhat unfortunate this is directly within the vertex manager. `BoundingBox` could have been a class with no static behavior, and just have an instance of itself situated within the OGL Renderer class otherwise.